src/app/layouts/common-layout/common-layout.routing.ts

```diff
@@ -17,7 +19,13 @@ export const CommonLayoutRoutes: Routes = [
   },
   { path: 'properties', component: PropertiesComponent },
   { path: 'no-properties', component: NoPropertiesComponent },
-  { path: 'create-company', component: CompanyComponent, canActivate: [AuthGuard] },
+  { path: 'no-deposits', component: NoDepositsComponent },
+  {
+    path: 'create-company',
+    component: CompanyComponent,
+    canActivate: [AuthGuard]
+  },
   { path: 'registersuccess', component: RegistersuccessComponent },
   { path: 'profile', component: ProfileComponent, canActivate: [AuthGuard] },
+  { path: 'deposits', component: DepositsComponent }
```

Comment:
Hey @Genza999, we need to protect our financial information from users that should not see them, let's protect this component with an AuthGuard, Also we will create another Guard to restrict this route to only me and not just anyone that's authenticated.

Reply:
Thanks @CryceTruly I have worked on this and also there is no way one would access another users deposits since only deposits of the logged-in user are returned back